### PR TITLE
Initial website scaffold

### PR DESCRIPTION
**Pull request by PaulWagener**
Creates initial HTML & CSS

### CODE REVIEW

**Comment by PaulWagener** [comment]
That `Set theme jekyll-theme-cayman` commit was necessary in my fork enable GitHub Pages. See also [this comment](https://github.community/t5/GitHub-Pages/Github-Pages-are-not-published-after-following-the-documentation/m-p/18676#M1398).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+    <details class="in-progress">
+        <summary>
+            <h2>De app is veilig</h2>
+            <span>Er is een adviescommissie</span>
+        </summary>
+
+        <div>
+            <ul>
+                <li>controle door NCSC aan de hand van een set publieke criteria waarover publiek gerapoprteerd zal worden</li>
```
Comment (alloy) [inline index.html:69]:
```suggestion
                <li>Controle door NCSC aan de hand van een set publieke criteria waarover publiek gerapporteerd zal worden</li>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                <li>Open source / volledig inzicht in de applicatie en de server code; in principe kan een ieder zelf zijn of haar eigen versie van de app bouwen en (binnen de door de telefoon leveranciers gestelde grenzen) verifiëren dat deze app identiek is aan de gedistribueerde app en deze testen. Inclusief een process om gevonden problemen te rapporteren.</li>
```
Comment (alloy) [inline index.html:70]:
```suggestion
                <li>Open source / volledig inzicht in de applicatie en de server code; in principe kan eenieder zelf zijn of haar eigen versie van de app bouwen en (binnen de door de telefoon leveranciers gestelde grenzen) verifiëren dat deze app identiek is aan de gedistribueerde app en deze testen. Inclusief een process om gevonden problemen te rapporteren.</li>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                <li>Standaard zaken als pentest, security audit (met publieke rapporten).</li>
+                <li>Een "Responsible Disclosure" policy met voldoende capaciteit, budget en kunde om gevonden en geporteerde problemen na eerste release aan te pakken</li>
```
Comment (alloy) [inline index.html:72]:
```suggestion
                <li>Een "Responsible Disclosure" policy met voldoende capaciteit, budget en kunde om gevonden en gerapporteerde problemen na eerste release aan te pakken</li>
```

? Of daadwerkelijk ‘geporteerd’?

**Comment by PaulWagener** [comment]
Review is always welcome, but the content in this PR is just a placeholder copied from an earlier version from [this](https://docs.google.com/document/d/1xIgjBf6ESpIrk6Ye1T556RQKkFQNsVEGAXt6OwXIG98) Google Doc. The real content is still under development.
But I've applied your fixes anyway :)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="nl">
+<head>
+    <meta charset="UTF-8">
+    <title>COVID-19 Notificatie App Status</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (SvanBoxel) [inline index.html:6]: Nitpick: Do we want to add a meta description for the Googles and Bings of this world?

Comment (PaulWagener) [reply index.html:6]: Added a meta description tag in the head

**Comment by PaulWagener** [comment]
I've switched out the content with placeholder texts and Lorem Ipsum.

**Comment by bwbroersma** [comment]
Weird stuff about the Jekyll theme, not an issue for me:
image
And https://openstate.github.io/rdnewviewer/ does work.

I fixed the Jekyll in #3 and removed the theme.

BTW you can kind of test github-pages yourself with:
```bash
docker run -it --rm -v "$PWD":/usr/src/app -p "4000:4000" starefossen/github-pages
```

**Pull request merged**